src/pages/solution/adobe-campaign-classic/release-notes.md

```diff
@@ -8,6 +8,14 @@ keywords:
 
 # Release notes
 
+## January 19, 2024
+
+* Added support for out-of-the-box push notifications:
+  * Basic notifications, including remind me later and call-to-action functionality
+  * Manual and automatic image carousels of two to five items
```

Comment:
we should remove the maximum image mention

Reply:
updated to three